src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,31 +1,39 @@
 package edu.kis.vh.nursery;
 
 public class DefaultCountingOutRhymer {
-	private int[] numbers = new int[12];
-	public int total = -1;
+// TODO: check name.
+	private static final int FULL_STACK_VALUE = 11;

```

Comment:
This constant is related to STACK_CAPACITY . Changing only one leads to an error (inconsistency).